src/index.css

```diff
@@ -171,3 +172,26 @@ input[type=number]::-webkit-outer-spin-button {
   top: 2em;
   right: 0;
 }
+
+/* DOWNLOAD DATA BUTTON */
+#download_data_fixed {
+    transform: rotate(90deg);
+    /* background-color: red; */
+    background-color: #fff;
+    color: #1CB5AC;
+    border-left: solid 1px #1CB5AC;
+    border-top: solid 1px #1CB5AC;
+    border-right: solid 1px #1CB5AC;
+    position: fixed;
+    top: 20%;
```

Comment:
We'd discussed changing the horizontal alignment to vertical. I think it would be good to make it's top-most edge align with its previous topmost edge so that it doesn't go into the header.